test/index.ts

```diff
@@ -62,9 +62,9 @@ expect.type<boolean>(Boom.isBoom(error, 404));
 expect.type<boolean>(Boom.isBoom(Boom.boomify(error)));
 
 expect.error(Boom.isBoom(error, 'test'));
-expect.error(Boom.isBoom('error'));
-expect.error(Boom.isBoom({ foo: 'bar' }));
-expect.error(Boom.isBoom({ error: true }));
+expect.type<boolean>(Boom.isBoom('error'));
+expect.type<boolean>(Boom.isBoom({ foo: 'bar' }));
+expect.type<boolean>(Boom.isBoom({ error: true }));
```

Comment:
These should be moved up a few lines, after the other non-error tests.

Also, would it work if the `expect.type<boolean>` is changed to `expect.type<false>`?

Reply:
Unfortunately no :(

![image](https://user-images.githubusercontent.com/92363/100614414-657e3880-3316-11eb-8679-03c70d4d64a9.png)